jaxley/modules/base.py

```diff
@@ -2272,67 +2284,68 @@ def _set_externals_in_view(self):
                 self.externals[name] = data[in_view]
                 self.external_inds[name] = inds_in_view
 
-    def _set_trainables_in_view(self):
-        trainable_inds = self.base.indices_set_by_trainables
-        trainable_inds = (
-            np.unique(np.hstack([inds.reshape(-1) for inds in trainable_inds]))
-            if len(trainable_inds) > 0
-            else []
-        )
-        trainable_node_inds_in_view = np.intersect1d(
-            trainable_inds, self._nodes_in_view
-        )
+    def _filter_trainables(
+        self, is_viewed: bool = True
+    ) -> Tuple[List[np.ndarray], List[Dict]]:
+        """filters the trainables inside and outside of the view
```

Comment:
Sentence case please.

Reply:
Also, could you add a sentence to the docstring about where this is used? I always find it hard to have these hidden methods where it is not immediately clear why they are needed.